### PR TITLE
Adding boilerplate for stu3 capability statement

### PR DESCRIPTION
**Pull request by Robert-W**
This PR adds the necessary boilerplate for generating a capability statement for STU3. Currently resources are returned as an empty array because they are only pulled in when they are available and have defined their own conformance. So as we add resources, we need to pull in their information from a `conformance.js` file in their own directory.

### CODE REVIEW

File: src/server/stu3/metadata/capability.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+	status: STU3.STATUSES.DRAFT,
+	date: moment().tz('America/New_York').format(),
+	publisher: 'Not provided',
+	kind: 'instance',
+	software: {
+		name: 'FHIR Server',
+		version: '0.0.1'
+	},
+	implementation: {
+		description: 'FHIR Test Server (STU#)'
```
Comment (jonterrylee) [inline src/server/stu3/metadata/capability.js]: STU#?

Comment (Robert-W) [reply src/server/stu3/metadata/capability.js]: lol, i'll update this

File: src/server/stu3/metadata/services/metadata.service.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
+const path = require('path');
+const glob = require('glob');
+const { VERSIONS } = require('../../../../constants');
+const { makeStatement, securityStatement } = require('../capability');
+
+// Glob for discovering all dstu2 conformance statements
```
Comment (jonterrylee) [inline src/server/stu3/metadata/services/metadata.service.js]: stu3

Comment (Robert-W) [reply src/server/stu3/metadata/services/metadata.service.js]: fixed

File: src/constants.js [hunk 3]
```diff
@@ -16,6 +16,25 @@ exports.DSTU2 = {
 	}
 };
 
+exports.STU3 = {
+	RESOURCE_TYPES: {
+		OPERATIONOUTCOME: 'OperationOutcome',
+		CONFORMANCE: 'CapabilityStatement',
+		OBSERVATION: 'Observation',
+		PATIENT: 'Patient'
```
Comment (andrewmarcus) [inline src/constants.js]: I don't like having to define these here.  I don't want to have to auto-generate the elements in this file to match the auto-generated classes.

**Pull request merged**